src/Actions/ManagesContacts.php

```diff
@@ -83,6 +101,36 @@ public function findOrCreateContact($email, $firstName, $lastName, $orgid = null
         return $this->createContact($email, $firstName, $lastName, $orgid);
     }
 
+    /**
+     * Updates a contact.
+     *
+     * @param Contact|int|string $id
+     * @param string|null        $email
+     * @param string|null        $firstName
+     * @param string|null        $lastName
+     * @param null               $orgid
+     *
+     * @return Contact|null
+     */
+    public function updateContact($id, $email, $firstName, $lastName, $orgid = null)
+    {
+        $id = $this->getContactId($id);
+
+        $this->put('contacts/'.$id, ['json' => ['contact' => compact('email', 'firstName', 'lastName', 'orgid')]]);
+
+        return $this->findContactById($id);
+    }
+
+    /**
+     * Deletes a contact.
+     *
+     * @param Contact|int|string $contact
+     */
+    public function deleteContact($contact)
+    {
+        $this->delete('contacts/'.$this->getContactId($contact));
```

Comment:
Be aware that on concatenation we add spaces around the dot. Also see other comments about the  getContactId() function. I would suggest:

`$this->delete('contacts/' . $contact->id);`

Reply:
I did this initially, but your style integration told me off so I removed the spacing.

Reply:
o realy? I am sorry. I will look into this myself. Keep it the way the style integration let it passes.